static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -0,0 +1,127 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import {getGroupDetailsQueryData} from 'sentry/views/issueDetails/utils';
+
+import type {TimelineEvent} from './useTraceTimelineEvents';
+
+interface TraceIssueEventProps {
+  event: TimelineEvent;
+}
+
+export function TraceIssueEvent({event}: TraceIssueEventProps) {
+  const organization = useOrganization();
+  const {projects} = useProjects({
+    slugs: [event.project],
+    orgId: organization.slug,
+  });
+  const project = projects.find(p => p.slug === event.project);
+  const issueId = event['issue.id'];
+  // We are view issue X and loading data for issue Y. This means that this call will not be cached,
+  // thus, it is a little bit slow to render the component.
+  // XXX: Create new endpoint that only fetches the metadata for the group
+  const {data: groupData, isLoading: isLoadingGroupData} = useApiQuery<Group>(
+    [
+      `/organizations/${organization.slug}/issues/${issueId}/`,
+      {query: getGroupDetailsQueryData({})},
+    ],
+    {
+      staleTime: 30000,
+      cacheTime: 30000,
+      retry: false,
+    }
+  );
+
+  // If any of data fails to load, we don't want to render the component
+  // Only "One other issue appears in the same trace. View Full Trace (X issues)" would show up
+  return (
+    <Fragment>
+      {isLoadingGroupData ? (
+        <LoadingIndicator mini />
+      ) : (
+        groupData && (
+          <TraceIssueLinkContainer
+            to={{
+              pathname: `/organizations/${organization.slug}/issues/${issueId}/events/${event.id}/`,
+              query: {
+                referrer: 'issues_trace_issue',
+              },
+            }}
+            onClick={() => {
+              trackAnalytics('issue_details.issue_tab.trace_issue_clicked', {
+                organization,
+                event_id: event.id,
+                group_id: issueId,
+              });
+            }}
+          >
+            {project && (
+              <ProjectBadge
```

Comment:
When `project` does not exist, can you render a placeholder in its place of the same size (`<Placeholder />`)? That way it won't pop in if it needs to be fetched

Reply:
Added locally.